airflow/models/renderedtifields.py

```diff
@@ -191,3 +179,20 @@ def delete_old_records(
             ]
 
             session.query(cls).filter(and_(*filter_tis)).delete(synchronize_session=False)
+
+    @retry_db_transaction
+    @classmethod
+    def remove_old_rendered_ti_fields_mysql(cls, dag_id, session, task_id, tis_to_keep_query):
```

Comment:
```suggestion
    @classmethod
    @retry_db_transaction
    def remove_old_rendered_ti_fields_mysql(cls, dag_id, session, task_id, tis_to_keep_query):
```
We need to change order as this won't work:

```
 $ airflow db check
  Traceback (most recent call last):
    File "/usr/local/bin/airflow", line 33, in <module>
      sys.exit(load_entry_point('apache-airflow', 'console_scripts', 'airflow')())
    File "/opt/airflow/airflow/__main__.py", line 40, in main
      args.func(args)
    File "/opt/airflow/airflow/cli/cli_parser.py", line 47, in command
      func = import_string(import_path)
    File "/opt/airflow/airflow/utils/module_loading.py", line 32, in import_string
      module = import_module(module_path)
    File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
    File "<frozen importlib._bootstrap>", line 994, in _gcd_import
    File "<frozen importlib._bootstrap>", line 971, in _find_and_load
    File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
    File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
    File "<frozen importlib._bootstrap_external>", line 678, in exec_module
    File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
    File "/opt/airflow/airflow/cli/commands/db_command.py", line 24, in <module>
      from airflow.utils import cli as cli_utils, db
    File "/opt/airflow/airflow/utils/db.py", line 27, in <module>
      from airflow.jobs.base_job import BaseJob  # noqa: F401
    File "/opt/airflow/airflow/jobs/__init__.py", line 19, in <module>
      import airflow.jobs.backfill_job
    File "/opt/airflow/airflow/jobs/backfill_job.py", line 29, in <module>
      from airflow import models
    File "/opt/airflow/airflow/models/__init__.py", line 30, in <module>
      from airflow.models.renderedtifields import RenderedTaskInstanceFields
    File "/opt/airflow/airflow/models/renderedtifields.py", line 36, in <module>
      class RenderedTaskInstanceFields(Base):
    File "/opt/airflow/airflow/models/renderedtifields.py", line 184, in RenderedTaskInstanceFields
      @classmethod
    File "/opt/airflow/airflow/utils/retries.py", line 96, in retry_db_transaction
      return retry_decorator(_func)
    File "/opt/airflow/airflow/utils/retries.py", line 55, in retry_decorator
      func_params = signature(func).parameters
    File "/usr/local/lib/python3.6/inspect.py", line 3065, in signature
      return Signature.from_callable(obj, follow_wrapped=follow_wrapped)
    File "/usr/local/lib/python3.6/inspect.py", line 2815, in from_callable
      follow_wrapper_chains=follow_wrapped)
    File "/usr/local/lib/python3.6/inspect.py", line 2193, in _signature_from_callable
      raise TypeError('{!r} is not a callable object'.format(obj))
  TypeError: <classmethod object at 0x7fd198edf9e8> is not a callable object
```

Reply:
Yep. I noticed yesterday but was too late to fix :)